projects/ui/src/temp-overrides.scss

```
@@ -5,53 +5,55 @@
-
-  // Vertical-nav not selected
-  --cds-alias-object-interaction-vertical-navigation-background-color: #{tokens.$cds-global-color-construction-100};
-  --cds-alias-object-interaction-vertical-navigation-background-hover: #{tokens.$cds-global-color-construction-200};
-  --cds-alias-object-interaction-vertical-navigation-background-active: #{tokens.$cds-global-color-construction-300};
-
-  // Vertical-nav selected
-  --cds-alias-object-interaction-vertical-navigation-background-selected: #{tokens.$cds-global-color-white};
-  --cds-alias-object-interaction-vertical-navigation-background-selected-hover: #{tokens.$cds-global-color-construction-200};
-  --cds-alias-object-interaction-vertical-navigation-background-selected-active: #{tokens.$cds-global-color-construction-300};
-  --cds-alias-object-container-background-inverse: #{tokens.$cds-global-color-construction-600};
-  --cds-alias-object-container-background-inverse-tint: #{tokens.$cds-global-color-construction-800};
-  --cds-alias-object-container-background-inverse-shade: #{tokens.$cds-global-color-construction-500};
-}
-
-[cds-theme~='dark'] {
-  --cds-alias-utility-gray: #{tokens.$cds-global-color-construction-300};
-
-  --cds-alias-object-container-backdrop-background: #{tokens.$cds-alias-object-opacity-300};
-  --cds-alias-object-interaction-background-hover: #{tokens.$cds-global-color-construction-800};
-  --cds-alias-object-interaction-background-active: #{tokens.$cds-global-color-construction-600};
-  --cds-alias-object-interaction-background-selected: #{tokens.$cds-global-color-blue-900};
-  --cds-alias-object-interaction-background-selected-active: #{tokens.$cds-global-color-construction-600};
-  --cds-alias-object-interaction-background-selected-hover: #{tokens.$cds-global-color-construction-700};
-
-  // Vertical-nav not selected
-  --cds-alias-object-interaction-vertical-navigation-background-color: #{tokens.$cds-global-color-construction-900};
-  --cds-alias-object-interaction-vertical-navigation-background-hover: #{tokens.$cds-global-color-construction-800};
-  --cds-alias-object-interaction-vertical-navigation-background-active: #{tokens.$cds-global-color-construction-600};
-
-  // Vertical-nav selected
-  --cds-alias-object-interaction-vertical-navigation-background-selected: #{tokens.$cds-global-color-blue-900};
-  --cds-alias-object-interaction-vertical-navigation-background-selected-hover: #{tokens.$cds-global-color-construction-800};
-  --cds-alias-object-interaction-vertical-navigation-background-selected-active: #{tokens.$cds-global-color-construction-600};
-  --cds-alias-object-container-background-inverse: #{tokens.$cds-global-color-construction-200};
-  --cds-alias-object-container-background-inverse-tint: #{tokens.$cds-global-color-construction-100};
-  --cds-alias-object-container-background-inverse-shade: #{tokens.$cds-global-color-construction-300};
-  --cds-alias-object-container-background-dark: #{tokens.$cds-global-color-construction-700};
+:root {
```

```suggestion

:root {
```

missing blank line here